test/Makefile

```diff
@@ -81,6 +81,7 @@ endif
 include ../src/osmodel.mak
 
 export OS
+BUILD=release
```

Comment:
Why is this necessary? `debug` is a better choice for testing in general and tests for specific optimization arguments, such as `-release`, do so via `PERMUTE_ARGS`. This variable affects only the test runner, right?